lib/src/models/player.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018, chances. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+part of spotify;
+
+@JsonSerializable()
+class Player extends Object {
+  Player() {}
+  factory Player.fromJson(Map<String, dynamic> json) =>
+      _$PlayerFromJson(json);
+  int timestamp;
```

Comment:
Please add the documentation comments from the [Spotify docs](https://developer.spotify.com/documentation/web-api/reference/player/get-the-users-currently-playing-track/) like seen in the other model definitions in this library.

Reply:
Added :white_check_mark: